app/controllers/webchat.js

```diff
@@ -33,6 +34,21 @@ export default Ember.Controller.extend({
 	field3value: '',
 	customAttributes: [],
 	error: '',
+	customPlugin: undefined,
+	environmentTld: computed('deployment', function() {
+		return purecloudEnvironmentTld();
+	}),
+	webChatVersion: computed('deployment', 'webChatService.deploymentCount', function() {
+		const deploymentId = this.get('deployment');
+		const deployment = this.get('deployments').find((d) => d.id === deploymentId);
+		return !deployment || !deployment.isV2 ? '1' : '2';
```

Comment:
This seems to be incorrect. If deployment is `undefined` from the line above then this line returns `true`. Do you mean `deployment || !deployment.isV2 ? '1' : '2'`?

Reply:
`!deployment` is intentional. If no deployment is selected, it defaults to displaying v1 data. That also prevents an NPE from the second part of the condition if deployment is undefined.